tests/terraform/checks/resource/aws/test_ElasticCacheHasSecurityGroup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import unittest
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.checks.resource.aws.ElasticacheHasSecurityGroup import check
+from checkov.terraform.runner import Runner
+
+
+class TestElasticacheHasSecurityGroup(unittest.TestCase):
+    def test(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_ElasticacheHasSecurityGroup"
+        report = runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        failing_resources = {
+            "aws_elasticache_security_group.exists",
+        }
+
+        failed_check_resources = set([c.resource for c in report.failed_checks])
```

Comment:
```suggestion
        failed_check_resources = {c.resource for c in report.failed_checks}
```
it does the same, but is faster, because it doesn't create a list first